spec/models/spree/credit_card_spec.rb

```diff
@@ -31,8 +31,8 @@ def stub_rails_env(environment)
 
         it "should be true if payment is checkout" do
           payment = build_stubbed(:payment, created_at: Time.zone.now)
-          allow(payment).to receive_messages :pending? => false,
-                                             :checkout? => true
+          allow(payment).to receive_messages pending?: false,
+                                             checkout?: true
```

Comment:
having the `?` at the end of the key looks odd but it's valid so 👍

Reply:
yes, these are methods from state_machine to check if the state is in a given point like pending? these will only be keys here where we mock the calls to these methods.